applications/nrf5340_audio/src/bluetooth/bt_management/controller_config/bt_mgmt_ctlr_cfg.c

```diff
@@ -158,7 +173,20 @@ int bt_mgmt_ctlr_cfg_version_get(uint16_t *ctlr_version)
 
 	struct bt_hci_rp_read_local_version_info *rp = (void *)rsp->data;
 
-	*ctlr_version = sys_le16_to_cpu(rp->hci_revision);
+	if (print_version) {
+		if (rp->manufacturer == COMPANY_ID_PACKETCRAFT) {
+			/* NOTE: The string below is used by the Nordic CI system */
+			LOG_INF("Controller: LL_ACS_NRF53. Version: %d", rp->hci_revision);
+		} else if (rp->manufacturer == COMPANY_ID_NORDIC) {
+			LOG_INF("Controller: SoftDevice: Version %s (0x%02x), Revision %d",
+				ver_str(rp->hci_version), rp->hci_version, rp->hci_revision);
+		} else {
+			LOG_ERR("Unsupported controller");
+			return -EPERM;
```

Comment:
I don't think we should return an error code here, since that would mean the watchdog would stop if you were using ZLL.

Reply:
Discussed offline. Will keep as is